pkgs/servers/akkoma/pleroma-fe/default.nix

```diff
@@ -0,0 +1,86 @@
+{ lib
+, stdenv
+, fetchFromGitea, fetchYarnDeps
+, fixup_yarn_lock, yarn, nodejs
+, jpegoptim, oxipng, nodePackages
+}:
+
+stdenv.mkDerivation rec {
+  pname = "pleroma-fe";
+  version = "2022-09-10";
+
+  src = fetchFromGitea {
+    domain = "akkoma.dev";
+    owner = "AkkomaGang";
+    repo = "pleroma-fe";
+    rev = "d7499a1f91fb5e4ec947445c566f8ae41cec6c72";
+    hash = "sha256-I+d+oTOTNaMPkk9vdFHhm/g8EtPTZnsWv8yPUa0uNHQ=";
+  };
+
+  offlineCache = fetchYarnDeps {
+    yarnLock = src + "/yarn.lock";
+    hash = "sha256-UkSdVRqFp6bj7zz4ZRMD47tVpuS7x7F1MiEei+87kyg=";
+  };
+
+  nativeBuildInputs = [
+    fixup_yarn_lock
+    yarn
+    nodejs
+    jpegoptim
+    oxipng
+    nodePackages.svgo
+  ];
+
+  # Build scripts assume to be used within a Git repository checkout
+  patchPhase = ''
+    sed -E -i \
+      -e '/^let commitHash =/,/;$/clet commitHash = "${builtins.substring 0 7 src.rev}";' \
```

Comment:
I personally remove the line with git rev-parse too in my derivation, although it probably isn’t an issue

Reply:
This expression should remove and replace all lines from `let commitHash =` until the first semicolon at the end of a line, replacing the complete statement.